geo/Cargo.toml

```diff
@@ -40,6 +40,7 @@ rand_distr = "0.4.3"
 
 ### boolean-ops test deps
 wkt = "0.10.1"
+serde_json = "1.0"
```

Comment:
Can you use WKT for your test fixtures? We already have the dependency and IMO it's easier to read as a human.

Reply:
Yeah that makes total sense. I'll change that :+1: